src/main/java/com/celements/search/web/classes/WebSearchClassPackage.java

```diff
@@ -12,7 +12,7 @@
 @Component(WebSearchClassPackage.NAME)
 public class WebSearchClassPackage extends AbstractClassPackage {
 
-  public static final String NAME = "webSearch";
+  public static final String NAME = "websearch";
```

Comment:
The package/collection "webSearch" is in use since two years.
See /celements-search/src/main/java/com/celements/search/web/classes/WebSearchClasses.java
The move from the ClassCollections to the Packages must not change the package/collection name. It is used in the XWikiPreferences and in "Activated Class Collections" and must work identically for collection and package.